sealer/cmd/root.go

```diff
@@ -51,28 +51,21 @@ func Execute() {
 
 func init() {
 	cobra.OnInitialize(initConfig)
-	rootCmd.PersistentFlags().StringVar(&rootOpt.cfgFile, "config", "", "config file (default is $HOME/.sealer.yaml)")
+	rootCmd.PersistentFlags().StringVar(&rootOpt.cfgFile, "config", "", "config file (default is $HOME/.sealer.json)")
```

Comment:
I think we don't need to add a config file, just save logs in ${rootfs}/var/log/xxx.log.

Reply:
If the configuration file does not exist, the default configuration is used. The default log file path is similar to /var/log/sealer/2006-01-01.log